pkg/azureclients/armclient/util.go

```diff
@@ -120,6 +120,8 @@ func DoHackRegionalRetryForGET(c *Client) autorest.SendDecorator {
 
 			bodyString := string(bodyBytes)
 			trimmed := strings.TrimSpace(bodyString)
+			klog.V(5).Infof("Send.sendRequest got response with ContentLength %d, StatusCode %d and responseBody length %d", response.ContentLength, response.StatusCode, len(trimmed))
```

Comment:
I got lots of such messages in csi driver logs now, is this really helpful? @feiskyer 
```
I1021 11:03:25.717281       1 util.go:123] Send.sendRequest got response with ContentLength -1, StatusCode 200 and responseBody length 1236
I1021 11:03:25.739747       1 util.go:123] Send.sendRequest got response with ContentLength -1, StatusCode 200 and responseBody length 12
I1021 11:03:25.754262       1 util.go:123] Send.sendRequest got response with ContentLength -1, StatusCode 200 and responseBody length 1237
```

Reply:
This would be useful when there is something wrong with ARM requests. Seems "ContentLength -1" is happening very regularly